### ISSUE TITLE
Comments and docstring different colors?

### ISSUE THREAD

**Issue by jabranham** [nashamri/spacemacs-theme#85]
Right now, comments and docstrings are the same color. I'd like to make them a little easier to distinguish between, though. Is that a PR you'd be interested in?

**Comment by nashamri** [comment]
Yeah sure, maybe a little variant of the comment's color could be used to change the docstrings, or anything, I'm open for suggestions, I did a quick check and there seem to be these three faces:

```
font-lock-comment-delimiter-face
font-lock-comment-face
font-lock-doc-face
```
I'm not sure what the `font-lock-comment-delimiter-face` supposed to be, though. I think we should leave the comments as they are and change the docstrings (the `font-lock-doc-face`).

**Comment by jabranham** [comment]
OK, great. Those are the three faces I found as well.

font-lock-comment-delimiter-face is used for the characters that start a comment (so `;` in emacs-lisp mode, for example). I think we should probably just leave that alone so it matches font-lock-comment-face (which it inherits from).

Also agree about leaving comments and changing docstrings. How about if we change it to green (currently #67b11d in GUI dark)? That's kinda in-between comment (blueish) and strings (which are greener), which seems reasonable to me? I think that would just involve changing the `:foreground ,comment` in line 164 to `:foreground ,green`.

**Comment by nashamri** [comment]
Or maybe we could just use the `meta` color for the docstrings (https://github.com/nashamri/spacemacs-theme/blob/master/spacemacs-common.el#L121). It's already adjusted for the light and dark versions and for the terminal, and the 'docstrings' can be thought of as meta information, so we got the semanitcs there also :smile: .what do you think?

**Comment by jabranham** [comment]
Sounds great!

**Comment by nashamri** [comment]
fixed by 5bcac14ab4b99bc3bef20fd62ef981c4590c4021
